packages/react-percy-test-framework/src/Suite.js

```diff
@@ -67,26 +78,47 @@ export default class Suite {
     return this.options;
   }
 
-  async getSnapshots() {
```

Comment:
There's no longer a `getSnapshots` function that returns all snapshots the user defined. Instead, we only ever need to render a single snapshot at a time, which happens client-side in Percy.